_posts/2015-10-12-omero-figure-demo-app-how-it-works.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "OMERO.figure demo app: How it works and how to use it"
+date:   2015-10-12
+---
+
+If you want to try using OMERO.figure without installing the app or the OMERO server,
+you can simply visit the demo site at [figure.openmicroscopy.org/demo/](http://figure.openmicroscopy.org/demo/)
+
+So how does the demo work, and what OMERO server does it use?
+
+<h2>OMERO.figure overview</h2>
+
+OMERO.figure is basically a bunch of html and javascript that runs in your browser, getting
+data and images from an OMERO server.
+
+It can also use the OMERO server to save the figure file and export the figure as PDF or TIFF files.
+
+
+<div class="panel panel-default">
+  <div class="panel-body" style="padding:0; text-align: center">
+    <img src="{{ site.baseurl }}/images/blog-omero-figure.png" style="width:600px"/>
+  </div>
+  <div class="panel-heading">
+    <h3 class="panel-title">OMERO.figure reads image data from OMERO, and can save figure files and PDFs</h3>
+  </div>
+</div>
+
+However, OMERO.figure doesn't **have** to get data from it's own "host" OMERO server. It
+can get data from any OMERO server that is available on the internet.
+
+<div class="panel panel-default">
+  <div class="panel-body" style="padding:0; text-align: center">
+    <img src="{{ site.baseurl }}/images/blog-demo-figure.png" style="width:600px"/>
+  </div>
+  <div class="panel-heading">
+    <h3 class="panel-title">The demo app reads image data from the OMERO-based JCB dataviewer</h3>
+  </div>
+</div>
+
+The demo app doesn't have it's own OMERO server. The html and javascript simply come from the 
+figure.openmicroscopy web site (which is hosted by github). The demo app instead
```

Comment:
I think I'll just remove the mention of github - Just confusing for most users and not needed.